### PR TITLE
Add rating extra api

### PR DESCRIPTION
**Pull request by riwaazBSijapati**

### CODE REVIEW

File: rating-service/src/main/java/edu/miu/ratingservice/service/impl/RatingServiceImpl.java [hunk 1]
```diff
@@ -40,12 +35,22 @@ public RatingServiceImpl(RatingRepository ratingRepository, UserClient userClien
 
     @Override
     public List<Rating> getRatingByDailyMealId(Long id) {
-        return ratingRepository.findByDailyMealId(id);
+        List<Rating> rating = ratingRepository.findByDailyMealId(id);
+        if (rating.size() != 0) {
+            return ratingRepository.findByDailyMealId(id);
```
Comment (ricardoianelli) [inline rating-service/src/main/java/edu/miu/ratingservice/service/impl/RatingServiceImpl.java]: Why calling it again?

File: rating-service/src/main/java/edu/miu/ratingservice/service/impl/RatingServiceImpl.java [hunk 2]
```diff
@@ -40,12 +35,22 @@ public RatingServiceImpl(RatingRepository ratingRepository, UserClient userClien
...
+            throw new MealNotFoundException();
+        }
     }
 
     @Override
     public List<Rating> getRatingByUserId(Long id) {
-        return ratingRepository.findByUserId(id);
+        List<Rating> rating = ratingRepository.findByUserId(id);
+        if (rating.size() != 0) {
+            return ratingRepository.findByUserId(id);
```
Comment (ricardoianelli) [inline rating-service/src/main/java/edu/miu/ratingservice/service/impl/RatingServiceImpl.java]: Same as last comment

File: rating-service/src/main/java/edu/miu/ratingservice/service/impl/RatingServiceImpl.java (hunk 1 above)
Comment (riwaazBSijapati) [reply rating-service/src/main/java/edu/miu/ratingservice/service/impl/RatingServiceImpl.java]: sorry can u repeat the question

Comment (riwaazBSijapati) [reply rating-service/src/main/java/edu/miu/ratingservice/service/impl/RatingServiceImpl.java]: Ohh i see what you mean, error adjusted

**Pull request closed without merging**